wgpu-types/src/openxr.rs

```diff
@@ -0,0 +1,24 @@
+/// Config
+pub struct OpenXROptions {
+
+}
+
+impl Default for OpenXROptions {
+    fn default() -> Self {
+        OpenXROptions {}
+    }
+}
+
+/// FIXME FIXME FIXME is this okay?
+pub struct OpenXRHandles {
```

Comment:
These OpenXR concepts are required at engine-space (bevy) in order to communicate with openxr. This is one take at transferring them between